.changeset/wet-camels-tell.md

```diff
@@ -0,0 +1,6 @@
+---
+"@fuel-ts/account": patch
+"@fuel-ts/crypto": patch
```

Comment:
```suggestion
"@fuel-ts/crypto": patch
"@fuel-ts/contract": patch
```

Reply:
@petertonysmith94 The contract package was not modified, only one test file that belongs to it.

Reply:
My mistake, you are correct.